otvoreni_akti/apps/scraper/tests/test_scrape_utils_html.py

```diff
@@ -33,7 +33,7 @@ def test_function_extracts_correct_data_from_page_with_hidden_redirect(self):
 
 class TestParseSubjectList(TestCase):
```

Comment:
This is good. I suggest you take a look how much easier writing tests as functions with pytest can be now that you've done it "the classical way" https://docs.pytest.org/en/latest/getting-started.html#getstarted

Reply:
I like using the Django TestCase. I can try direct asserts too but without pytest they aren't that informative so I avoided using them.

Reply:
OK, that's perfectly fine of course. Just thought I'd share it with you 🙂